lib/ProductOpener/Ingredients.pm

```diff
@@ -3512,6 +3543,35 @@ sub get_missing_ecobalyse_ids ($ingredients_ref) {
 	return @ingredients_without_ecobalyse_ids;
 }
 
+=head2 get_geographical_area ($country_code)
+
+Retrieve the geographical area for ecobalyse. (NOTE : this is a first version that'll soon be improved)
+
+=head3 Arguments
+
+=head4 $country_code
+
+reference to the name of the country 
+
+=head3 Return values
+
+=head4 $ecobalyse_area
+
+=cut
+
+sub get_geographical_area ($country_code) {
```

Comment:
We use country code to refer to the 2 letters country codes, could you rename $country_code to $originid ? in the origins field you will get things like "en:france" or "en:britanny".